frontend/version.lua

```diff
@@ -86,4 +92,22 @@ function Version:getBuildDate()
     return self.date
 end
 
+--- Updates `VERSION_LOG_FILE` and keep the file small
+-- @string model device model (may contain spaces)
+function Version:updateVersionLog(current_model)
+    local last_line = LogFile:getLastLineAndShrinkFile(VERSION_LOG_FILE, MAX_NB_LOG_LINES - 1)
```

Comment:
You wrote your logfile.lua module when it was needed for the InfoMessage log if they were saved to disk.
I don't think it is necessary any longer, and probably not for this version log: if you would write a line of 100 bytes each day, that would make a file of 36500 bytes each year, so 365 KB 10 years later :) which I think is bearable - no need to arbitrarily rotate/shrink this file, the user may be happy knowing in 2028 he installed KOReader in 2023.